app/ConfigValidation/ValidateConfigApplication.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\ConfigValidation;
+
+use Symfony\Component\Console\Application;
```

Comment:
Took me some time to realize that the Application keyword here is from the Console component. I first thought it was the Silex one.